runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -1448,16 +1448,7 @@ private HiscoreLookup getCorrectLookupFor(final ChatMessage chatMessage)
 			return new HiscoreLookup(localPlayer.getName(), hiscoreEndpoint);
 		}
 
-		// Public chat on a leagues world is always league hiscores, regardless of icon
-		if (chatMessage.getType() == ChatMessageType.PUBLICCHAT || chatMessage.getType() == ChatMessageType.MODCHAT)
```

Comment:
Mods chat on leagues uses the pmod icon, so it may be worth leaving this in?

Reply:
Per discord discussion, leaving this check in